server/datastore/psql/create_maintenance_requests.go

```diff
@@ -0,0 +1,33 @@
+package psql
+
+import (
+    "time"
+    "github.com/a2fumn2022/gpmn-nsl/server/datastore"
+)
+
+func (db *DB) CreateMaintenanceRequests(request string, userID int, time time.Time) (int, error) {
```

Comment:
Rather than taking in each individual param, why not take in a `MaintenanceRequest` struct? That's simpler and makes it easier to update in the future.

Reply:
fixed: used datastore.MaintenanceRequest struct as param